resources/variants/gmax15plus_12_e3d.inst.cfg

```diff
@@ -1,5 +1,5 @@
 [general]
-name = 0.25mm E3D (Difficult)
+name = 1.2mm E3D Volcano
```

Comment:
It's not a good idea to rename variants since then you need to also modify the VersionUpgrade to convert all the all names to the new names. If there is no other possibility, then please add this modification to your PR.

Reply:
Ah. I just copied the 0.25mm file as a starting point to make a new 1.2mm file (which didn't exist before). I did not realize it would count as "renaming" the variant. It must have happened because we removed the 0.25mm file which we will restore per your suggestion below.